ci/centos7-ci.sh

```diff
@@ -44,10 +44,7 @@ install_dependencies() {
     yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
     yum install -y openresty-pcre-devel openresty-zlib-devel
 
-    export runtime_version=${APISIX_RUNTIME}
-    wget "https://raw.githubusercontent.com/api7/apisix-build-tools/apisix-runtime/${APISIX_RUNTIME}/build-apisix-runtime.sh"
-    chmod +x build-apisix-runtime.sh
-    ./build-apisix-runtime.sh latest
```

Comment:
Can this be reverted now?

Reply:
Not yet, I need a new version of `build-apisix-runtime.sh` after merging https://github.com/api7/apisix-build-tools/pull/376